specification/eventhub/resource-manager/Microsoft.EventHub/stable/2017-04-01/EventHub.json

```diff
@@ -2503,6 +2534,47 @@
         }
       },
       "description": "The response of the List MessagingRegions operation."
+    },
+	"MessagingPlanproperties": {
```

Comment:
>MessagingPlanproperties [](start = 2, length = 23)

this entire type is only used as service return type. Please mark each property as "readOnly": true